### PR TITLE
preview: get maximum file size from config

### PR DESCRIPTION
**Pull request by mdonadoni**
Closes reanahub/reana#728

### CODE REVIEW

File: reana-ui/src/pages/workflowDetails/components/filePreview/FilePreview.js [hunk 1]
```diff
@@ -151,9 +151,9 @@ function checkConstraints(fileName, size) {
   if (!match) {
     const fileExt = fileName.split(".").pop();
     content = `${fileExt} files cannot be previewed. Please use download.`;
-  } else if (size > FILE_SIZE_LIMIT) {
+  } else if (size > sizeLimit) {
     content = `File size is too big to be previewed (limit ${
-      FILE_SIZE_LIMIT / 1024 ** 2
+      sizeLimit / 1024 ** 2
```
Comment (giuseppe-steduto) [inline reana-ui/src/pages/workflowDetails/components/filePreview/FilePreview.js:156]: We may want to round this number to show only one or two decimal digits at max - if `config.filePreviesSizeLimit` is not a multiple of a MiB, the result of the division may show a lot of decimal places.

**Review by giuseppe-steduto — COMMENTED** [review]
Looks good! Just left a small comment on the formatting of the message in case the file is too big to be previewed.

File: reana-ui/src/pages/workflowDetails/components/filePreview/FilePreview.js (hunk 1 above)
Comment (mdonadoni) [reply reana-ui/src/pages/workflowDetails/components/filePreview/FilePreview.js]: I have updated the PR to show the limit in human-readable format, basically the same as [`_human_readable_bytes`](https://github.com/reanahub/reana-db/blob/02c0e4a93c46bfaa302dd7d84647a92d1673e8c2/reana_db/models.py#L1018) from reana-db

File: reana-ui/src/util.js [hunk 2]
```diff
@@ -237,6 +237,34 @@ export function formatSearch(term) {
...
+ * @param {Number} size File size in bytes.
+ * @returns The formatted human-readable file size.
+ */
+export function formatFileSize(size, digits = 2) {
+  if (size === 0) {
+    return "0 Bytes";
+  }
+
+  const units = [
+    "Bytes",
```
Comment (giuseppe-steduto) [inline reana-ui/src/util.js:251]: very minor: what about removing the capital letter (`bytes`) or maybe using directly the symbol `B`?

**Review by giuseppe-steduto — APPROVED** [review]
LGTM!

File: reana-ui/src/util.js (hunk 2 above)
Comment (mdonadoni) [reply reana-ui/src/util.js:251]: I also like `bytes` or `B` more, but I kept `Bytes` to be consistent with reana-db:
```python
    @staticmethod
    def _human_readable_bytes(bytes_):
        """Convert bytes usage to human readable string."""
        if bytes_ == 0:
            return "0 Bytes"
        units = ["Bytes", "KiB", "MiB", "GiB", "TiB", "PiB", "EiB", "ZiB", "YiB"]
        ...
```

**Comment by mdonadoni** [comment]
Rebased and merged!

**Pull request merged**